backend/tests/Unit/Monolog/SensitiveDataProcessorTest.php

```
@@ -0,0 +1,192 @@
+        self::assertSame(SensitiveDataProcessor::REDACTED, $context['dropbox_refresh_token']);
+    }
+
+    public function testRedactsAuthorizationHeadersAndCookies(): void
+    {
+        $context = $this->process([
+            'Authorization' => 'Bearer abcdefghijklmnop',
+            'cookie' => 'PHPSESSID=deadbeefdeadbeef',
+            'Set-Cookie' => 'XSRF-TOKEN=abc123',
+        ]);
+
+        self::assertSame(SensitiveDataProcessor::REDACTED, $context['Authorization']);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Unit\Monolog\SensitiveDataProcessorTest::assertSame().


(staticMethod.notFound)